SketchUp/Geom/PolygonMesh.rb

```diff
@@ -51,20 +51,71 @@ class Geom::PolygonMesh
   def add_point(point)
   end
 
-  # The {#add_polygon} method is used for adding a polygon to a polygon mesh.
-  #
-  # @example
-  #   mesh = Geom::PolygonMesh.new
-  #   point1 = Geom::Point3d.new(0, 1, 2)
-  #   point2 = Geom::Point3d.new(1, 0, 2)
-  #   point3 = Geom::Point3d.new(2, 0, 1)
-  #   index = mesh.add_polygon(point1, point2, point3)
-  #
-  # @param [Array<Integer>] points
-  #   A list of points that represent the
-  #   polygon's vertices.
-  #
-  # @return [Integer] the index of the polygon in the mesh
+  # The {#add_polygon} method is used for adding a {#polygons polygon} to a
+  # {PolygonMesh}. All variations of this method require at least 3 elements
```

Comment:
I'm ok either way - though I prefer fully qualified names when we do reference something in the reference brackets.